### ISSUE TITLE
Unable to upload backup v0.23.8

### ISSUE THREAD

**Issue by Salman9000** [pocketbase/pocketbase#6152]
I download the backup from my production environemnt which is running pocketbase 0.23.8.
![image](https://github.com/user-attachments/assets/d3e4bad1-a122-4a3d-ade9-ac5302696b6b)

Then i download pocketbase exe v0.23.8 and started it using pocketbase serve in my windows. Note I also tried the same in WSL as well. Then I upload the backup.
![image](https://github.com/user-attachments/assets/33e70f4a-e7e0-4101-8eaa-aedfb275b2da)

It gives error "Something went wrong" with no logs on my terminal. What to do?
![image](https://github.com/user-attachments/assets/b2efc404-e525-49c3-abfc-74bc52bc903a)

**Comment by ganigeorgiev** [comment]
It could be a platform specific issue, because I'm not able to reproduce it on Linux.

Please try to run the executable with `--dev` and check if there are errors printed in the console.

**Comment by ganigeorgiev** [comment]
You can also manually extract the backup and replace your local `pb_data` with the one from the archive, but please check first for the error with `--dev` as it could be hiding a bug.

**Comment by Salman9000** [comment]
Ok using --dev i can see the output. It says "entity too large" 
![image](https://github.com/user-attachments/assets/75bd75cc-6f85-485c-ae32-7c492bbb5ce7)
However my backup file is of 49mb

**Comment by ganigeorgiev** [comment]
Oh, I see. This is because with v0.23 we have a default request body size limit set to ~32mb.

I'll add exception for the backups upload endpoint with the next minor release a little bit later today.

As a temp workaround, you can manually extract the archive and replace your `pb_data` directory (make sure that the application is not running).

**Comment by Salman9000** [comment]
Oh okay makes sense. May know why did you limit it to 32mb? Couldnt the backups very easily exceed this limit?
Secondly i replaced the contents inside the pb_data with those of the archive. It was success. However when i login to my pocketbase it gets stuck on loading. 
![image](https://github.com/user-attachments/assets/ed03f14b-a66e-4e8d-baa7-b276de02c06a)
It is trying to send an email but it never does and is just completely stuck. Also i dont think it should send email using resend when in loggin into pb admin.

**Comment by ganigeorgiev** [comment]
The 32mb limit is the default that it is applied for all endpoints (as mentioned previously for the collection record file uploads this limit is dynamic and based on the specific field limits) and it exists to minimize the risk of abusing the API.

Regarding "_when i login to my pocketbase it gets stuck on loading_" I don't know what do you mean by this. It can result in an error, Yes, but it shouldn't "stuck" whatever you mean by this. The email send alert functionality is optional and can be changed from the `_superusers` auth collection options in the dashboard. The email send itself could fail because most likely you don't have `sendmail` installed or SMTP configured on the running platform, but again, it shouldn't "stuck".

**Comment by Salman9000** [comment]
Yes it was just taking alot of time to load. I removed the smtp and now it is logging in instantly.

**Comment by ganigeorgiev** [comment]
I've submitted a fix in `master` to skip the default body limit for the backup upload endpoint. It'll be available together with other minor fixes with the v0.23.12 sometime later this week.

Regarding the SMTP slow response - the underlying mail library at the moment doesn't have option to pass timeout or custom cancellation context. I'll try to create a PR in the upstream library for that and later I'll define a default timeout for the mail send to minimize the wait time. Again, you can disable the email alerts from the `_superusers` auth collection options and the "Send email alert for new login" option (every auth collection has such option).

**Comment by reisfmb** [comment]
@ganigeorgiev 

I was getting the same problems on 0.23.1. I've now updated to 0.24.0 and I'm getting a new problem when trying to upload a 34mb backup:

![Image](https://github.com/user-attachments/assets/4345e827-5151-47a3-bc2a-e327f2105660)

Any idea why?!

**Comment by ganigeorgiev** [comment]
@reisfmb You can start the executable with `--dev` and/or check for the failed request in the _Dashboard > Logs_ for more detailed error messages.

If you have a reverse-proxy (nginx, apache, caddy, etc.) make sure that it is configured properly and that it doesn't interfere with the request.

More than this I don't know how to help further.

**Comment by reisfmb** [comment]
@ganigeorgiev Thx for the quick response.

![Image](https://github.com/user-attachments/assets/151f8760-fc67-4578-baf6-4458aa9bcf14)

That's what I got from the logs, which is weird because I surely uploaded the .zip file...

**Comment by ganigeorgiev** [comment]
@reisfmb Again, without any information about your environment or steps to reproduce I'm not sure how to help you. I've tested it locally and it works correctly for me. We also have automated API tests that should verify the correct behavior: https://github.com/pocketbase/pocketbase/blob/37ff943f6704b8d55390847114a9cc23badc0632/apis/backup_test.go#L218.

**Comment by reisfmb** [comment]
The only additional information I have for now is that the pb instance is being hosted on the railway free plan, deployed with the following Dockerfile

```

FROM alpine:3 as downloader

ARG TARGETOS
ARG TARGETARCH
ARG VERSION=0.24.0

ENV BUILDX_ARCH="${TARGETOS:-linux}_${TARGETARCH:-amd64}"

RUN apk add --no-cache \
    ca-certificates \
    unzip \
    wget \
    zip \
    zlib-dev

RUN wget https://github.com/pocketbase/pocketbase/releases/download/v${VERSION}/pocketbase_${VERSION}_${BUILDX_ARCH}.zip \
    && unzip pocketbase_${VERSION}_${BUILDX_ARCH}.zip \
    && chmod +x /pocketbase

COPY ./pb_hooks /pb_hooks
COPY ./pb_migrations /pb_migrations

FROM scratch

EXPOSE 8080

COPY --from=downloader /pocketbase /usr/local/bin/pocketbase
COPY --from=downloader /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
COPY --from=downloader /pb_hooks /pb_hooks
COPY --from=downloader /pb_migrations /pb_migrations

CMD ["/usr/local/bin/pocketbase", "serve", "--dev", "--http=0.0.0.0:8080", "--dir=/pb_data", "--publicDir=/pb_public", "--hooksDir=/pb_hooks", "--migrationsDir=/pb_migrations"]
```

**Comment by ganigeorgiev** [comment]
Please test it locally just PocketBase, without Docker or reverse proxy.

Additionally I recommend to bump your version to the latest [v0.24.4](https://github.com/pocketbase/pocketbase/releases/tag/v0.24.4) as it comes with some minor security and bug fixes.

If when testing locally you are still experiencing the same error, try to open the browser's _Insect tool > Network tab_ and check what is the body that is submitted with the backup upload request.
